app/views/games/play.html.slim

```diff
@@ -11,3 +11,10 @@ br
 = link_to 'Undo Round', undo_round_game_path(@game), class: 'btn btn-primary', method: 'delete'
 br
 = link_to 'Finish Game', finish_game_path(@game), class: 'btn btn-primary', method: 'patch'
+
+coffee:
+  ($ window).keypress (e) ->
+    if e.keyCode == 0 or e.keyCode == 32
+      e.preventDefault()
+      ($ "#next-button").click()
```

Comment:
While you would definitely write `$ '#next-button'` if there was no chained method call, I disagree that you should put the opening parenthesis before the `$` instead of after.  In other languages that don't require parenthesis, like Ruby, you wouldn't wrap the whole expression in parenthesis.  For example, if you wanted to find an object with id 1, you would write `Object.find 1`.  However, if you then wanted to access an attribute on the resulting object, you would write `Object.find(1).attribute`, not `(Object.find 1).attribute`.